package.json

```diff
@@ -22,231 +41,27 @@
     "beautifier"
   ],
   "categories": [
-    "Other"
+    "Formatters",
+    "Linters"
   ],
   "activationEvents": [
-    "onCommand:remark.reformat",
     "onLanguage:markdown"
   ],
-  "main": "./out/extension.js",
-  "contributes": {
```

Comment:
will there be some commands or configuration that will still be wanted/needed after the update?

Reply:
I don’t think any custom commands or configuration options are wanted at this moment.

Perhaps we can add support for custom languages (mdx) once https://github.com/unifiedjs/unified-engine/issues/56 has been resolved.

Maybe autofixing appears to be quite complex and may require a command registration, but that’s not yet implemented by the language server.